devtools/conda-envs/environment.yaml

```diff
@@ -0,0 +1,13 @@
+name: membrane-curvature
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python
+  - pip
+  - numpy
+  - MDAnalysisTests
```

Comment:
I added MDAnalysisTests because the Usage includes examples uses it, so better to have them in the environment.

Reply:
Maybe stick it in pip with 2.0.0b0, as I think it's required to match the MDAnalysis and MDAnalysisTests versions

Reply:
I would avoid MDAnalysisTests as a default dependency, it's 100+ MB of disk space which can be a lot.

We often use MDAnalysisTests as part of the core library usage examples, but don't make folks install it by default. What I recommend here is maybe just adding an extra few lines in the setup instructions telling folks they can install MDAnalysisTests if they want to follow the usage instructions?

Reply:
Thanks, that's a good point. I will leave MDAtests out for now. I added the instructions to install tests but in the Documentation section, which may appear too late in the README. I will reorganize it.